src/main/java/frc/robot/Robot.java

```diff
@@ -132,25 +132,25 @@ public void teleopPeriodic() {
     HashMap<String, InputContainer<?>> inputMap = hardwareInterface.getInputValueMap();
     RobotModel model = controller.run(inputMap);
 
-    List<DeviceOutputCommand> shooterCommands = shooterSubsystem.run(model.shooterModel.orElse(new ShooterSubsystemModel(ShooterSubsystemModel.ShooterState.STOPPED)));
-    List<DeviceOutputCommand> conveyorCommands = conveyorSubsystem.run(model.conveyorModel.orElse(
-        new ConveyorSystemModel(
-          ConveyorSystemModel.IntakeState.STOPPED,
-          ConveyorSystemModel.IntakePosition.UP,
-          ConveyorSystemModel.ShooterBlockState.CLOSE
-        )
-    ));
-    List<DeviceOutputCommand> hangerCommands = hangerSubystem.run(model.hangerModel.orElse(new HangerSystemModel(HangerSystemModel.HangerState.STOPPED)));
+    // List<DeviceOutputCommand> shooterCommands = shooterSubsystem.run(model.shooterModel.orElse(new ShooterSubsystemModel(ShooterSubsystemModel.ShooterState.STOPPED)));
```

Comment:
Lets uncomment these lines before merging, it is not correct and also causing test failures